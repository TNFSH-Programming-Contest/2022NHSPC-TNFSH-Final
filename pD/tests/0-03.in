20 10
1 2
1 3
1 4
2 5
2 6
3 10
4 13
4 14
6 7
6 8
10 11
4 15
4 16
8 9
11 12
16 17
16 18
16 19
17 20
15 19 16
1 12 1
20 20 20
7 7 8
1 8 3
5 20 2
2 9 6
9 12 1
9 12 2
9 12 3
