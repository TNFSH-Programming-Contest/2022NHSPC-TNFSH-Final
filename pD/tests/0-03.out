4
16
20
0
0
5
2
10
2
1
